prime-router/docs/playbooks/add_ip_blacklist.md

```diff
@@ -0,0 +1,54 @@
+# Add IP to blacklist
```

Comment:
Can we use "deny list" instead of "blacklist"? I think we're trying to get away from white/black in terms of lists, and allow/deny is more descriptive :-)